README.md

```diff
@@ -48,6 +48,17 @@ are identical and can't get mangled by a broken deps installation.
 
 We're using webpack to build a JS bundle.
 
+### Hot module reloading
+
+If you run the webpack-dev-server you can get hot module reloading.
+
+For example run `grunt serve`
+
+And then visit http://localhost:8080/webpack-dev-server/management.html
```

Comment:
As discussed, do the simplest thing first and land it :) Looking at the CORS cookie thing though, I think we can run the dev server on the host and talk to `pay.dev/api/` with just a few setting changes in the service.